src_Core/Core/CoreW.bsv

```diff
@@ -110,9 +119,37 @@ import DM_CPU_Req_Rsp ::*;
 // ================================================================
 // The Core module
 
-(* synthesize *)
-module mkCoreW #(Reset dm_power_on_reset)
-               (CoreW_IFC #(N_External_Interrupt_Sources));
+typedef WindCoreMid #( // AXI lite subordinate control port parameters
+                       21, 32, 0, 0, 0, 0, 0
```

Comment:
What are 21 and 32 here?

Reply:
address width and data width into the axi4 lite "management" slave